src/propagators.jl

```diff
@@ -135,8 +142,15 @@ function propagate(prob::QuKrylov, op, t, current_t, current_qustate)
     return next_state
 end
 
+function propagate(prob::QuODESolvers, op, t, current_t, current_qustate)
+    next_state = prob.method((t,y)-> -im*coeffs(op)*y, coeffs(current_qustate), [current_t, t], points=:specified,
```

Comment:
If you use the last tagged version of ODE.jl (0.2) the keyword `:specified` is only supported with `ode23s` as far as I remember. This means that you should use `[2][end]` instead of `[2][2]` (and that a lot of allocations will happen). We should probably encourage people to use the development version of ODE via `ODE.checkout()`.